docs/getting-started-guides/mesos.md

```diff
@@ -80,41 +78,31 @@ Replace `<mesos_master>` with the url of the Mesos master. Depending on your Mes
 Now start the kubernetes-mesos API server, controller manager, and scheduler on a Mesos master node. Replace `<mesos_master>` with the Mesos master URL used above in the `mesos-cloud.conf`:
 
 ```bash
-$ bin/km apiserver \
+$ km apiserver \
   --address=${servicehost} \
-  --etcd_servers=http://${servicehost}:4001 \
+  --etcd-servers=http://${servicehost}:4001 \
   --service-cluster-ip-range=10.10.10.0/24 \
   --port=8888 \
-  --cloud_provider=mesos \
-  --cloud_config=mesos-cloud.conf \
+  --cloud-provider=mesos \
+  --cloud-config=mesos-cloud.conf \
   --v=1 >apiserver.log 2>&1 &
 
-$ bin/km controller-manager \
+$ km controller-manager \
   --master=$servicehost:8888 \
-  --cloud_provider=mesos \
-  --cloud_config=./mesos-cloud.conf  \
+  --cloud-provider=mesos \
+  --cloud-config=./mesos-cloud.conf  \
   --v=1 >controller.log 2>&1 &
 
-$ bin/km scheduler \
+# NOTE: <mesos_master> here should match the one in your mesos-cloud.conf file
+$ km scheduler \
   --address=${servicehost} \
-  --mesos_master=<mesos_master> \
-  --etcd_servers=http://${servicehost}:4001 \
-  --mesos_user=root \
-  --api_servers=$servicehost:8888 \
+  --mesos-master=<mesos_master> \
```

Comment:
I would just have them define `export MESOS_MASTER=<hostname or ip>` up top and use `${MESOS_MASTER}` here, rather than mixing env vars with manual replacements.